### PR TITLE
Added logging on associating a pipeline artifact

### PR DESCRIPTION
**Pull request by owenhuynMSFT**
**Problem Statement**
There was an issue reported to Azure DevOps detailing how a pipeline artifact was capitalized after publishing even though one's YAML file had a lower cased artifact name. Since we weren't able to reproduce the issue, we decided to add some logging to try to understand why the issue was happening for some users.

### CODE REVIEW

File: src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV1.cs [hunk 1]
```diff
@@ -63,6 +63,14 @@ public class PublishPipelineArtifactTaskV1 : PipelineArtifactTaskPluginBaseV1
             CancellationToken token)
         {
             string artifactName = context.GetInput(ArtifactEventProperties.ArtifactName, required: false);
+            if (string.IsNullOrEmpty(artifactName))
+            {
+                context.Output($"Artifact name was not inserted for publishing.");
+            }
+            else
+            {
+                context.Output($"Inputted artifact name: {artifactName}");
```
Comment (fadnavistanmay) [inline src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV1.cs]: Single t.

**Comment by jtpetty** [comment]
LGTM - the one failing test is flaky and I have started investigating it

**Comment by jtpetty** [comment]
I am not sure why I cannot merge this as it passed https://dev.azure.com/mseng/PipelineTools/_build/results?buildId=10502026&view=results

@owenhuynMSFT  - can you merge with master and push?

**Pull request merged**